sql-plugin/src/main/scala/com/nvidia/spark/rapids/AstUtil.scala

```diff
@@ -119,4 +122,112 @@ object AstUtil {
       }
     }
   }
+
+  abstract class NonAstJoinCondSplitStrategy() {
+    def remainedCond(): Option[Expression]
```

Comment:
A name like `astCondition` might be better since it represents the condition that can be evaluated via AST.

Reply:
Updated.